docs/source/usage/plugins/energyHistogram.rst

```diff
@@ -15,28 +15,48 @@ There are several command line parameters that can be used to set up this plugin
 Replace the prefix ``e`` for electrons with any other species you have defined, we keep using ``e`` in the examples below for simplicity.
 Currently, the plugin can be set *once for each species*.
 
-=========================================== ======================================================================
+=========================================== =====================================================================================
 PIConGPU command line option                description
-=========================================== ======================================================================
+=========================================== =====================================================================================
 ``--e_energyHistogram.period``              The ouput periodicity of the **electron** histogram.
                                             A value of ``100`` would mean aoutput at simulation time step *0, 100, 200, ...*.
                                             If set to a non-zero value, the energy histogram of all **electrons** is computed.
                                             By default, the value is ``0`` and no histogram for the electrons is computed.
+``--e_energy.filter``                       Use filtered particles. All available filters will be shown with ``picongpu --help``
```

Comment:
hm, the user never interacts in our workflows with the `picongpu` binary and a reference to `picongpu --help` could be done on *any* of these files. I recommend to instead reference `particleFilters.param` here:

```
Available filters are set up in :ref:`particleFilters.param <usage-params-core>`.
```